### ISSUE TITLE
const/override binary expression clarification.

### ISSUE THREAD

**Issue by Kangz** [gpuweb/gpuweb#4557]
For example for division of signed integer there is the following:
<img width="465" alt="image" src="https://github.com/gpuweb/gpuweb/assets/472810/a2df48dd-f3ed-405d-b3b1-fb3bb14bb8a2">

It is unclear what should happen if "e1" is a const expression and `e2` a pipeline overridable expression because const expressions are overridable expressions. I suggest rewording with trailing otherwises.

 - If e1 is most negative value in T, and e2 is -1:
   - It is a shader-creation error if e1 and e2 are const-expressions, otherwise
   - It is a pipeline-creation error if e1 and e2 are both override-expressions, otherwise
   - e1

**Comment by teoxoy** [comment]
> It is unclear what should happen if "e1" is a const expression and `e2` a pipeline overridable expression because const expressions are overridable expressions.

My reading was that it should be a pipeline-creation error (since const-expressions are also override-expressions).

**Comment by Kangz** [comment]
Ah yes that case works, but const + const falls in both first conditions.

**Comment by dj2** [comment]
I believe that then gets covered by `Each requirement [will](https://www.w3.org/TR/WGSL/#behavioral-requirements) be checked at the earliest opportunity.` From https://www.w3.org/TR/WGSL/#errors
